From 95921ccf77ae037bb3b95f45a632f60975c6c9bb Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Tue, 11 Feb 2020 17:03:18 -0500
Subject: [PATCH] mpmdlif.c: don't suppress kernel <linux/types.h> inclusion

Defining _LINUX_TYPES_H before including <linux/keystone_remoteproc.h> causes
<linux/types.h> to skip defining kernel types, resulting in undefined __u32

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 src/daemon/mpmdlif.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/daemon/mpmdlif.c b/src/daemon/mpmdlif.c
index 0c31604..1abd00f 100644
--- a/src/daemon/mpmdlif.c
+++ b/src/daemon/mpmdlif.c
@@ -46,7 +46,6 @@
 #include <byteswap.h>
 #include <stdint.h>
 
-#define _LINUX_TYPES_H
 #include "linux/keystone_remoteproc.h"
 #include "internal/rsc_types.h"
 //#include <elf.h>
-- 
2.7.4

